package.json

```diff
@@ -17,7 +17,7 @@
     "devDependencies": {
         "@0xproject/utils": "^0.1.0",
         "async-child-process": "^1.1.1",
-        "ethereumjs-testrpc": "6.0.3",
+        "ethereumjs-testrpc": "^6.0.3",
```

Comment:
Hm, should we specify the specific minor and patch version we want or do we trust `TestRPC` to properly adhere to semantic versioning?

Reply:
I've added `--frozen-lockfile` flag so as long as someone doesn't manually update it we'll stay on that version

Reply:
That flag only applies for CI runs and not when people are running the tests locally. I'd prefer we explicitly version.

Reply:
Not really. We have a yarn.lock in the repo and unless someone force-updates or removes the file the version will stay locked. That's the whole point of yarn. Deterministic builds.